src/BulkManager/BulkAction/EditHandler.php

```diff
@@ -107,7 +107,7 @@ public function getI18nLabel()
      */
     public function Link($action = null)
     {
-        return Controller::join_links(parent::Link(), $this->stat('url_segment'), $action);
+        return Controller::join_links(parent::Link(), static::config()->url_segment, $action);
```

Comment:
What is the `stat` method? What are the implications of not calling that method anymore?

Reply:
isn't the stat method an old ss3 thing that allowed to access static variables? at any rate the intent here is clear: get the static url_segment depending on the class or subclass

Reply:
> isn't the stat method an old ss3 thing that allowed to access static variables?

I don't know, hence why I'm asking what it is, and what the implications are of suddenly not calling it :p
I can't find a method called `stat` _anywhere_ in a full kitchen sink installation. I guess I'll just have to assume you're right that it's explicitly intended to only get the configuration.

Please change the syntax to `static::config()->get('url_segment')`